packages/admin/admin/src/form/fields/ToggleButtonGroupField.tsx

```diff
@@ -0,0 +1,20 @@
+import { Field, type FieldProps } from "../Field";
+import { FinalFormToggleButtonGroup, type FinalFormToggleButtonGroupProps } from "../FinalFormToggleButtonGroup";
+
+export type ToggleButtonGroupFieldProps<FieldValue> = FieldProps<FieldValue, HTMLSelectElement> & FinalFormToggleButtonGroupProps<FieldValue>;
+
+/**
+ * The `ToggleButtonGroupField` is a form field component intended to be used to switch between sections in a form.
+ *
+ * This is especially useful when the user needs to choose between different types
+ * of input for the same conceptual field — for example, selecting between an address or coordinate.
+ */
+export function ToggleButtonGroupField<FieldValue = unknown>({ options, optionsPerRow, ...restProps }: ToggleButtonGroupFieldProps<FieldValue>) {
+    return (
+        <Field {...restProps}>
+            {(fieldProps) => {
+                return <FinalFormToggleButtonGroup input={fieldProps.input} meta={fieldProps.meta} options={options} optionsPerRow={optionsPerRow} />;
```

Comment:
@nsams adjusted it, even though I don't see a big benefit in having this abstraction layer (`FinalFormXXX`) and having two components now and marking it anyways as deprecated..

Reply:
btw. i skipped the `componentProps` like we did it in SelectField because I think it is more convenient to have the props directly on the component. 

What do you think?

Reply:
> even though I don't see a big benefit in having this abstraction layer (FinalFormXXX)

I don't think there is any point anymore in having the additional `FinalFormXxx` components, if we have the `XxxField` component. They are just remaining from back when we didn't have `XxxField` components.

Reply:
Both have their use case:
- FinalFormXxx is the form input component compatible with final-form (has FieldRenderProps as props)
- XxxField is a covenience wrapper that renders a comet Field (not a final-form Field) and a FinalFormXxx into it
    - comet Field is a final-form wrapper that also renders a label, required sign, error messages and other stuff (renders FieldContainer)
    - XxxField is less code to write and is better typed BUT less flexible (due to the hardcoded Field->FieldContainer)

I think we should have both and deprecate none on them. And use in most cases XxxField.

Reply:
We'd need to revert https://github.com/vivid-planet/comet/pull/2426 then.

Reply:
oh, I wasn't aware that they got deprecated.

Maybe in practice 100% of all usages do actually need a label etc.

Reply:
Reverted in https://github.com/vivid-planet/comet/pull/3855.